scripts/promptflow-ci/promptflow_ci.py

```diff
@@ -165,7 +165,9 @@ def check_flow_run_status(
     parser.add_argument('--flow_submit_mode', type=str, default="sync")
     parser.add_argument('--run_time', type=str, default="default-mir")
     parser.add_argument('--skipped_flows', type=str,
-                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow")
+                        default="bring_your_own_data_qna,template_chat_flow,template_eval_flow,playground-ayod-rag,\
+                        chat-quality-safety-eval,qna-non-rag-metrics-eval,qna-quality-safety-eval,qna-rag-metrics-eval,\
```

Comment:
Please remove the spaces before chat-quality-safety-eval and rai-eval-ui-dag-flow, otherwise it will be counted as the flow name like below:
![image](https://github.com/user-attachments/assets/f100c97b-0d32-4a19-8526-c76b37104e0f)